src/main/kotlin/racingcar/model/CarCollection.kt

```diff
@@ -0,0 +1,63 @@
+package racingcar.model
+
+import racingcar.command.CommandInvoker
+import racingcar.command.DoRaceCommand
+import racingcar.dto.CarDto
+import racingcar.dto.WinnerDto
+
+
+class CarCollection(carNameString: String) {
+    private var cars : List<Car>
+
+    init {
+        val carNames = carNameString.split(",")
```

Comment:
`Constants.kt`에 선언한 `SEPARATOR`를 활용하시면 좋을 듯 합니다!

```suggestion
        val carNames = carNameString.split(SEPARATOR)
```

Reply:
처음에는 `SEPARATOR`를 썼는데, 오류 찾으면서 바꿨다가 못바꿨네요 ㅠㅠ 다음부터는 확실하게 관리를 해야겠네요!